be/src/runtime/exec_env_init.cpp

```diff
@@ -230,6 +230,17 @@ Status ExecEnv::_init_mem_env() {
     }
     int32_t index_percentage = config::index_page_cache_percentage;
     uint32_t num_shards = config::storage_page_cache_shard_size;
+    if ((num_shards & (num_shards - 1)) != 0) {
+        int old_num_shards = num_shards;
+        num_shards = 1;
+        while (num_shards < old_num_shards) {
+            num_shards <<= 1;
```

Comment:
But on BE, we have `BitUtil::RoundUpToPowerOfTwo` to complete the round

Reply:
done